# List of available singularity recipes
| Software                                                                                                          | Version         | Notes |
| ----------------------------------------------------------------------------------------------------------------- |:---------------:| ----- |
[alleleCount](https://github.com/cancerit/alleleCount) | v4.1.2 | |
[AmpliconArchitect](https://github.com/virajbdeshpande/AmpliconArchitect)                                           | v2              | |
[AmpliconClassifier](https://github.com/jluebeck/AmpliconClassifier) 												| v1			  | See recipe ampliconarchitect_ampliconclassifier|
[bamm](http://bamm-project.org/) | v2.5.0 | |
[bcftools](http://www.htslib.org/)                                                                                  | v.1.10.2        | |
[bedtools](https://github.com/arq5x/bedtools2/)                                                                     | v2.29.2         | |
[bwa](https://github.com/lh3/bwa)                                                                                   | v.0.7.17 (r1188)| Additives: samtools (v.1.10), picard (v2.23.3) |
[chasmPlus](https://github.com/KarchinLab/CHASMplus) | v1 | |
[CHISEL](https://github.com/raphael-group/chisel )                                                                  | v1              | Additives: python libraries (numpy,scipy,pandas,matplotlib,seaborn, setuptools), samtools(v.1.10), bcftools(v.1.10.2) |
[CN_learn](https://github.com/girirajanlab/CN_Learn) | v 5/10/2019 | |
[cnvkit](https://cnvkit.readthedocs.io/en/stable/) | vb218280e | |
[complex_timing]() |  | |
[crossmap](https://crossmap.readthedocs.io/en/latest/)                                                              | v.0.5.1         | |
[cutadapt](https://cutadapt.readthedocs.io/en/stable/)                                                              | v2.10           | |
[deeptools](https://deeptools.readthedocs.io/en/develop/) | v3.4.3  | |
[DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html)              														| v1.30.0 		  |	R package (R v.4.0.3) |
[driverpower](https://driverpower.readthedocs.io/en/latest/)                                                        | v1.0.2          | |
[EAGLE2](https://alkesgroup.broadinstitute.org/Eagle/)                                                              | v2.4.1          | |
[FASTQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)                                                | v.0.11.9        | See recipe ngs_qc |
[GATK](https://github.com/broadinstitute/gatk/)                                                                     | v.3.8-1-0 and v.4.1.8.1 | |
[GATK](https://github.com/broadinstitute/gatk/)                                                                     | v.4.1.8.1       | |
[GATK](https://storage.googleapis.com/gatk-software/package-archive/gatk/GenomeAnalysisTK-3.8-1-0-gf15c1c3ef.tar.bz2) | v.3.8-1-0     | |
[gridss](https://github.com/PapenfussLab/gridss) | v2.13.2 | |
[hla_hd](https://www.gckyoto.com/https/wwwgenomemedkyoto-uacjp/hla-hd) | v1 | |
[hmmcopy_utils](https://github.com/shahcompbio/hmmcopy_utils) | v1 | |
[HMMcopy](https://bioconductor.org/packages/release/bioc/html/HMMcopy.html) | v1.36.0  | |
[LARVA](http://larva.gersteinlab.org/)																				| v2.0			  | |  
[lohhla](https://bitbucket.org/mcgranahanlab/lohhla/src/master/) | v2 | |
[medicc2](https://bitbucket.org/schwarzlab/medicc2/src/master/) | v1 | |
[multiqc](https://multiqc.info/)                                                                                    | v.1.9           | See recipe ngs_qc |
[MuPeXI](https://github.com/ambj/MuPeXI) | v1.2.0 | |
[Mutect](https://software.broadinstitute.org/cancer/cga/mutect)														| v1.1.4 		  | See recipe polysolver_hg38 |
[mutpanning](https://github.com/vanallenlab/MutPanningV2)                                                           | v2              | |
[Novoalign](http://www.novocraft.com/products/novoalign/) 															| v3.09.05 		  | See recipe polysolver_hg38 |
[oncodriveclustl](https://bitbucket.org/bbglab/oncodriveclustl/src/master/)                                         | v1              | |
[oncodrivefml](https://bitbucket.org/bbglab/oncodrivefml/src/master/)                                               | v2              | |
[Picard](https://broadinstitute.github.io/picard/) 																	| v1.141          | See recipe polysolver_hg38 |
[polysolver](https://software.broadinstitute.org/cancer/cga/polysolver)												| v1			  | |
[PureCN](https://bioconductor.org/packages/release/bioc/html/PureCN.html) | v2.0.2 | |
[pyclone-vi](https://github.com/Roth-Lab/pyclone-vi/)															    | v.1.0.2         | |
[samtools](http://www.htslib.org/)                                                                                  | v1.10           | |
[sequenza](https://cran.r-project.org/web/packages/sequenza/vignettes/sequenza.html|v2019-05-09||
[sigprofilerjulia](https://bitbucket.org/bbglab/sigprofilerjulia/src/master/) | v1 | |
[SigProfilerMatrixGenerator](https://github.com/AlexandrovLab/SigProfilerMatrixGenerator) | v1.2 | |
[snpEff](http://pcingola.github.io/SnpEff/) | v5.1 | |
[sratoolkit](https://github.com/ncbi/sra-tools/)                                                                    | v2.10.8         | |
[STAR](https://github.com/alexdobin/STAR)																			| v.2.7.1a 	  | |
[STRELKA](https://github.com/Illumina/strelka)              														| v2.9.10 		  |	See recipe polysolver_hg38 |
[varscan2](http://varscan.sourceforge.net/) | v2.3.8 | |
[VEP, vcf2map]() | v105 | |
[whatshap](https://whatshap.readthedocs.io/en/latest/) | v1.2 | |
